### ISSUE TITLE
Error on EnVi Geometry export

### ISSUE THREAD

**Issue by mfloresVicomtech** [rgsouthall/vi-suite07#9]
Hi, I'm trying to simulate with E+, I've created a simple rectangular room with a window and all the surfaces with its corresponding EnVi material. Room object set as EnVi Surface. When I try to export the geometry Blender throws this error:
Warning: No mesh data to join
```
Error: Python: Traceback (most recent call last):
  File "C:\Users\mflores\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\vi-suite07-master\vi_operators.py", line 1948, in invoke
    pregeo(context, self)
  File "C:\Users\mflores\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\vi-suite07-master\envi_export.py", line 747, in pregeo
    node.zupdate(bpy.context)
  File "C:\Users\mflores\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\vi-suite07-master\vi_node.py", line 5085, in zupdate
    self.outputs[-1].viuid = '{}#{}'.format(obj.name, obj.data.polygon_layers_int["viuid"].data[bface.index].value)
AttributeError: 'Mesh' object has no attribute 'polygon_layers_int'
```

The geometry gets generated inside EN_COLLECTION and it seems to be ok, but if I try to export EnVi Context it then throws this error:
```
Error: Python: Traceback (most recent call last):
  File "C:\Users\mflores\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\vi-suite07-master\vi_operators.py", line 2042, in invoke
    error = enpolymatexport(self, eg_coll, node, locnode, envi_materials(), envi_constructions())
  File "C:\Users\mflores\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\vi-suite07-master\envi_export.py", line 106, in enpolymatexport
    en_idf.write(emnode.ep_write(mat.name, ln))
  File "C:\Users\mflores\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\vi-suite07-master\vi_node.py", line 7348, in ep_write
    mats = con_node.ec.propdict[con_node.envi_con_type][con_node.envi_con_list]
KeyError: 'Ceiling 1'
```

This only happens with zone type materials for interiors description, same geometry with materials for exterior works well.

> ViSuite 0.7 on Blender 4.0.1 on Windows 11 (but same thing happens on ViSuite 0.6.1 on Blender 2.93)

**Comment by rgsouthall** [comment]
I suspect it's because of API changes in 4.0. You may need to stick with 3.6 until I've fixed it.

**Comment by mfloresVicomtech** [comment]
> I suspect it's because of API changes in 4.0. You may need to stick with 3.6 until I've fixed it.

I've just tested it on Blender 3.6 with ViSuite 0.7 and it also raises the KeyError on EnVi context export. I leave here attached the project file for you to test:
[visuite_testing.zip](https://github.com/rgsouthall/vi-suite07/files/13456360/visuite_testing.zip)

**Comment by rgsouthall** [comment]
This should now be fixed in master for 4.0

**Comment by mfloresVicomtech** [comment]
Tested latest version it now works, thanks!
